buildscripts/checkstyle.xml

```diff
@@ -43,7 +43,7 @@
         </module>
         <module name="LineLength">
```

Comment:
Hmm - can we delete this? I think spotless already does the desired behavior, line length 100 except for long links

Reply:
As a principle I guess I'd remove any whitespace related checks from checkstyle since spotless should be handling

Reply:
does it handle markdown-style links, though? Check out the diff in the ResourceAttributes here. Would it handle that, I wonder?

Reply:
I think more precisely, what it does it break at 100, except when a single token is longer than 100 chars, so wouldn't matter what type of link it is. I guess it's pretty sane behavior since we generally don't have a token longer than 100 chars unless we really needed to have it.

Reply:
I really wish that the semconv yaml didn't have markdown in it, because we're just getting that from the generator.

Reply:
If someone has time, the generator could be adapted to render the markdown to HTML, fit for Javadoc.

Reply:
@anuraaga I finally see what you are saying...why are we checkstyling when we're already spotlessing? Great question. I'd be happy to remove this check and let spotless just do our formatting for us.